cli/src/ui.rs

```diff
@@ -590,29 +590,35 @@ impl Ui {
     }
 
     /// Repeat the given prompt until the input is one of the specified choices.
+    /// Returns the index of the choice.
     pub fn prompt_choice(
         &self,
         prompt: &str,
         choices: &[impl AsRef<str>],
-        default: Option<&str>,
-    ) -> io::Result<String> {
+        default_index: Option<usize>,
```

Comment:
Perhaps this should just be `usize`? Or do you think the behavior if this is `None` and `Self::can_prompt() == false` is reasonable?

Reply:
Ah, I realized we do have a use-case.

Reply:
`prompt_choice()` should fail in that case.